ci/build.py

```diff
@@ -90,15 +90,33 @@ def build_docker(platform: str, docker_binary: str, registry: str) -> None:
     #
     # This doesn't work with multi head docker files.
     # 
-    cmd = [docker_binary, "build",
-           "-f", get_dockerfile(platform),
-           "--build-arg", "USER_ID={}".format(os.getuid()),
-           "--build-arg", "GROUP_ID={}".format(os.getgid()),
-           "--cache-from", tag,
-           "-t", tag,
-           "docker"]
-    logging.info("Running command: '%s'", ' '.join(cmd))
-    check_call(cmd)
+
+    num_retries = int(os.environ['DOCKER_BUILD_NUM_RETRIES']) if 'DOCKER_BUILD_NUM_RETRIES' in os.environ else 1
```

Comment:
Would a --retry-actions argument for the script make sense here instead of an env var?

Reply:
I wanted to avoid adding a new argument here because Docker build failing is not a typical use-case a user should encounter. My preference is to keep the arguments as few as possible and abstract these CI-specific things into environment variables instead.

Reply:
Agree that we should abstract the CI specific things into env vars.  I'd argue it could be useful for users to be able to specify a retry count, so to me it seems it would be best fit to be an argument.  I could go either way on this one, what do you think @larroy ?